provisioning/provisioning-service-client/src/test/java/tests/unit/com/microsoft/azure/sdk/iot/provisioning/service/configs/EnrollmentGroupTest.java

```diff
@@ -161,6 +161,31 @@ private MockEnrollmentGroup makeMockedSymmetricKeyEnrollmentGroup()
                 new SymmetricKeyAttestation(VALID_PRIMARY_KEY, VALID_SECONDARY_KEY));
     }
 
+    private void setAllocationPolicyWorks(AllocationPolicy expectedAllocationPolicy, SymmetricKeyAttestation mockedAttestation)
```

Comment:
Are these supposed to be tests? you'll need the @ test annotation and to make them public if so

Reply:
This is a helper method used by the tests. It's factoring out common code used by a few number of tests.